docs/rules/no-duplicate-imports.md

```diff
@@ -2,6 +2,8 @@
 
 An ES6/ES2015 import can be spread over multiple lines, but this takes up unneeded whitespace. This rules validates that all imports from a single module exists in a single import statement.
```

Comment:
s/validates/requires/
add backticks to `import` statement at the end of the second sentence

In my first reading of the first sentence I took import as statement and lines as line breaks so my mind sort of rewrote it as I processed the meaning. Here is a rough draft of how I interpreted it:

Imports from an ECMAScript module can be spread over multiple `import` statements, …